packages/orbit-tailwind-preset/src/presets/foundation/theme/button.ts

```diff
@@ -0,0 +1,17 @@
+import { convertHexToRgba, defaultTokens } from "@kiwicom/orbit-design-tokens";
+
+export const opacity = {
+  "button-disabled": defaultTokens.buttonDisabledOpacity,
+};
+
+export const boxShadow = {
```

Comment:
This file and those tokens should be a part of the component preset, not the default one :)